a-star.ts

```diff
@@ -1,25 +1,29 @@
+//+array+SimpleLocation sim:7.5%, meowbit:10.5% ms comparing with origin
 namespace scene {

```

Comment:
```suggestion
namespace scene {
    //costs, scaled up by 1000
    NEIGHBOR_COST=1000
    DIAGONAL_COST=1414
```
add consts at beginning